.editorconfig

```diff
@@ -0,0 +1,220 @@
+# top-most EditorConfig file
+root = true
+
+# Set default behavior to:
+#   UTF-8 encoding,
+#   Unix-style line endings,
+#   a newline ending the file,
+#   4 space indentation,
+#   trimming of trailing whitespace
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+# Scripts files
+[*.{cmd,csproj,ps1,psm1,sh}]
+indent_size = 2
+
+# Solution files
+[*.sln]
+end_of_line = crlf
+indent_style = tab
+
+# Config files
+[*.{yml,yaml,json,config,props,targets,nuspec,resx}]
+indent_size = 2
+
+[*.{md,markdown}]
+trim_trailing_whitespace = false
+
+
+# CSharp coding styles
+
+[*.cs]
+# Indentation options
+# @see: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/formatting-rules#indentation-options
+csharp_indent_block_contents = true
+csharp_indent_braces = false
+csharp_indent_case_contents = true
+csharp_indent_case_contents_when_block = false
+csharp_indent_labels = one_less_than_current
+csharp_indent_switch_labels = true
+
+# Spacing options
+# @see: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/formatting-rules#spacing-options
+csharp_space_after_cast = false
+csharp_space_after_colon_in_inheritance_clause = true
+csharp_space_after_comma = true
+csharp_space_after_dot = false
+csharp_space_after_keywords_in_control_flow_statements = true
+csharp_space_after_semicolon_in_for_statement = true
+
+csharp_space_around_binary_operators = before_and_after
+csharp_space_around_declaration_statements = do_not_ignore
+
+csharp_space_before_colon_in_inheritance_clause = true
+csharp_space_before_comma = false
+csharp_space_before_dot = false
+csharp_space_before_open_square_brackets = false
+csharp_space_before_semicolon_in_for_statement = false
+
+csharp_space_between_empty_square_brackets = false
+csharp_space_between_method_call_empty_parameter_list_parentheses = false
+csharp_space_between_method_call_name_and_opening_parenthesis = false
+csharp_space_between_method_call_parameter_list_parentheses = false
+csharp_space_between_method_declaration_empty_parameter_list_parentheses = false
+csharp_space_between_method_declaration_name_and_open_parenthesis = false
+csharp_space_between_method_declaration_parameter_list_parentheses = false
+csharp_space_between_parentheses = false
+csharp_space_between_square_brackets = false
+
+# Wrap options
+# @see: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/formatting-rules#wrap-options
+csharp_preserve_single_line_blocks = true
+csharp_preserve_single_line_statements = true
+
+# CSharp styling options
+# @see: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/language-rules#c-style-rules
+csharp_prefer_braces = true:silent
```

Comment:
We do not allow if statements without braces in general so I think this can be upgrade from silent to error. If there are some instances where we missed then the severity can be upgraded to warning until such time that we have fixed all brace-less if statements.

Reply:
I've changed it to false and allow braces for multiline expression